cartographer/sensor/proto/sensor.proto

```diff
@@ -20,6 +20,15 @@ option java_outer_classname = "Sensor";
 
 import "cartographer/transform/proto/transform.proto";
 
+message RangefinderPoint {
+  transform.proto.Vector3f position = 1;
```

Comment:
I am not sure I like 'position' name here. In ROS it was called just a [point](http://docs.ros.org/melodic/api/sensor_msgs/html/msg/PointCloud.html).

Reply:
Sure. We're not bound by ROS conventions here, though. A `RangefinderPoint` having a `point` member sounds kind of odd to me.

Reply:
When I see `position` I always confuse it with `pose` which also has orientation. What about `x_y_z` or `coordinates`? Or leave it as `position`.

Reply:
At the robotics classes at my uni, we were taught the following definition of a robot [pose](https://en.wikipedia.org/wiki/Pose_(computer_vision)): _The combination of position and orientation is referred to as the pose of an object_.

So if that's okay with you, I'd leave it as position :-)

Reply:
Yeah, it is ok. :) I did not have robotics classes :)